Main.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-loop-func */
+let book = [];
+
+function CreateBook(titleValue, authorValue, id) {
+  this.title = titleValue;
+  this.author = authorValue;
+  this.id = id;
+}
+
+const pushBook = () => {
+  const titleValue = document.getElementById('Title').value;
+  const authorValue = document.getElementById('Author').value;
+  const id = book.length;
+  const theBook = new CreateBook(titleValue, authorValue, id);
+  book.push(theBook);
+};
```

Comment:
- [ ] Kindly add some validation measures to prevent empty inputs book objects from being submitted to the array. At this moment, empty inputs are allowed to submit. Kindly look into that and fix it.

[your-page.webm](https://user-images.githubusercontent.com/26597432/204518735-085f227c-58e4-40bf-b0eb-14b3bc12de46.webm)

Reply:
@Qoosim  please sir that is not part of the project requirements.